.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive",
+    "java.debug.settings.onBuildFailureProceed": true
+}
```

Comment:
Could this be added to the .gitignore. Don't think IDE settings and preferences should appear in our services